services/prompter.js

```diff
@@ -246,6 +246,10 @@ async function Prompter(program, requests) {
         name: 'appHostname',
         message: 'What\'s the IP/hostname on which your application will be running? ',
         default: 'localhost',
```

Comment:
Did you check the case if you set a remote environment with the `lumber generate` option (without prompt)?
Is the error well catched/displayed?

Reply:
No it's not, I have to fix this.

Reply:
Done !